src/Chapter4.hs

```diff
@@ -488,6 +495,15 @@ Implement the 'Applicative' instance for our 'List' type.
   may also need to implement a few useful helper functions for our List
   type.
 -}
+instance Applicative List where
+  pure :: a -> List a
+  pure x = Cons x Empty
+
+  (<*>) :: List (a -> b) -> List a -> List b
+  Empty <*> _ = Empty
+  _ <*> Empty = Empty
+  Cons f fs <*> l = append (fmap f l) (fs <*> l)
```

Comment:
Unfortunately, `append` is not a standard function, and `List` doesn't have any instances here in order to use any standard function.
You can implement `append` manually here 👍🏼 ➕